operator/runner/runner.go

```diff
@@ -203,6 +203,9 @@ func (r *Runner) Run(ctx context.Context, id int64) error {
 		}
 		return env
 	})
+	if err != nil {
+		return err
```

Comment:
for this particular error we should do the following to ensure it is reported back to the server and visible int he user interface:

```
return r.handleError(ctx, m.Stage, err)
```

Reply:
Done, @bradrydzewski . Do you guys prefer that I squash commits, or leave this as-is?